### DISCUSSION TITLE
Second commit doesn't show up in Pull Request

### DISCUSSION THREAD

**Discussion by Anton-V-K** [community/community#10767] [Pull Requests] [answered]
The scenarion (it used to work fine):
1. Fork, create branch, commit changes, push.
2. Submit PR.
3. Review.
4. Wait.
5. Make changes in the same branch, commit, push => You expect the recent commit is shown in the PR (created on the 2-nd step), but it doesn't (BUG).

Example PR: https://github.com/traccar/traccar-client-android/pull/430

Similar issue - https://github.community/t/pull-request-isnt-updating-with-new-commits/226239

**Comment by willsmythe** [comment, marked as answer]
Sorry for the inconvenience here. There was an incident that impacted several pull requests around this time. We are tracking a fix internally that will improve resiliency and close the gap that can allow this problem to occur.
